packages/jest-cli/src/cli/index.js

```diff
@@ -163,7 +161,7 @@ const buildArgv = (maybeArgv: ?Argv, project: ?Path) => {
     .options(args.options)
     .epilogue(args.docs)
     .check(args.check)
-    .version(false).argv;
+    .version(`v${VERSION}`).argv;
```

Comment:
without passing in the version, it printed `undefined`. Not sure why... Probably something weird with using the `./jest` script

Reply:
Have you tried this?

>If no arguments are passed to version (.version()), yargs will parse the package.json of your module and use its version value.

https://github.com/yargs/yargs/blob/master/docs/api.md#versionoption-description-version

Reply:
yeah, but that prints `undefined`. It seems like it looks up from `cwd` and uses the `package.json` there, instead of the closest local one.

This is only an issue when using `./jest` in this repo, it works doing `node package/jest-cli/bin/jest.js --version`